src/tasks/installers/operator.ts

```diff
@@ -307,6 +313,8 @@ export class OperatorTasks {
    */
   deleteTasks(flags: any): ReadonlyArray<Listr.ListrTask> {
```

Comment:
In deleteTasks we have to be compatible with previous versions.
1. Check if resource with namespace prefix exists and delete it if so
2. Otherwise, check if resource with a default name exists and delete it if so